`expiration_time`, `message`

makcc666 opened the pull request:
Добавлены опции к `SendOfferToSteamId` и `SendOffer`

makcc666 committed:
`expiration_time`, `message`

Добавлены опции к `SendOfferToSteamId` и `SendOffer`

feed4rz commented:
В API какой вход? Секунды? 

Вот [тут](https://github.com/OPSkins/trade-opskins-api/blob/master/ITrade/SendOffer.md#input) говорится, что они это **поменяют**, но сейчас используется _unix timestamp_.

Другой вопрос. Даже если на вход подаются секунды, то зачем ограничивать их на 10 символов?

makcc666 commented:
Да, в секундах. У них в описании ошибка.
10 символов - стандартная длина для подобного времени ( like php `time()` ).
Возможно когда они и поменяют действия опции `expiration_time`, сохранят распознавание даты.

По типу:
```js
expiration_date = trade.expiration_time.length === 10 ? trade.expiration_time : Math.floor(Date.now() / 1000) + trade.expiration_time; 
```

makcc666 committed:
send offers `expiration_time`

expiration time (minimum 120 seconds (2 minutes), max 14 days (1209600 seconds)

makcc666 reviewed (commented):
Изменён функционал времени жизни обмена.
expiration time (minimum 120 seconds (2 minutes), max 14 days (1209600 seconds)
